src/components/header/Header.jsx

```diff
@@ -149,45 +149,85 @@ const Header = () => {
         }
       >
         <div className="flex-row-center drawer-header">
-          <Link to="/products" className="nav-link">
```

Comment:
suggestion - Except innerText everything else is same can be mapped from list of a headings

Reply:
Yeah. I have made the changes. Thanks!